unity-renderer/Assets/DCLPlugins/DebugPlugins/ShapesBoundingBoxDisplayer/EntityWireframe.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using DCL.Models;
+using DCLPlugins.DebugPlugins.Commons;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+internal class EntityWireframe : IShapeListener
+{
+    private readonly GameObject wireframeOriginal;
+
+    private GameObject entityWireframe;
+
+    public EntityWireframe(GameObject wireframeOriginal)
+    {
+        this.wireframeOriginal = wireframeOriginal;
+    }
+
+    void IDisposable.Dispose()
+    {
+        CleanWireframe();
+    }
+
+    void IShapeListener.OnShapeUpdated(IDCLEntity entity)
+    {
+        entityWireframe ??= Object.Instantiate(wireframeOriginal);
+
+        Transform wireframeT = entityWireframe.transform;
+
+        wireframeT.position = entity.meshesInfo.mergedBounds.center;
+        wireframeT.localScale = entity.meshesInfo.mergedBounds.size * 1.01f;
```

Comment:
can we move this to a constant? This way we have more info about what is this magic number

Reply:
On another hand , we are not setting the rotation of the wireframe, if the entity is rotated in different axies, will it work?

Reply:
no need to set rotation there.
size and position are realted to the `mergedBounds`. After those lines the wireframe is added as child of the entity's `gameObject` so it will rotate with the entity